tests/integration/select/select_cached.js

```diff
@@ -22,7 +22,7 @@ QUnit.asyncTest( "dialog sized select should alter the value of its parent selec
 			resetHash,
 
 			function() {
-				$.mobile.changePage( "cached.html" );
+				$( ".ui-pagecontainer" ).pagecontainer( "change", "cached.html" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?